file_example.c

```diff
@@ -193,13 +194,26 @@ static int file_flush(struct tcmu_device *dev, struct tcmulib_cmd *cmd)
 	return 0;
 }
 
+static int file_reconfig(struct tcmu_device *dev, struct tcmulib_cfg_info *cfg)
+{
+	switch (cfg->type) {
+	case TCMULIB_CFG_DEV_SIZE:
+		return tcmu_update_num_lbas(dev, cfg->data.dev_size);
```

Comment:
You probably want to edit the check_state in the txt file to check for INQUIRY/GET_EVENT_STATUS_NOTIFICAITON/REPORT_LUNS(tcmu doesnt handle this atm but w/e), request sense....
Something like this:
        if(dev->flags & TCMU_CAPACITY_CHANGED &&
           cdb[0] != INQUIRY &&
           cdb[0] != GPCMD_GET_EVENT_STATUS_NOTIFICATION) {
                ret = tcmu_set_sense_data(cmd->sense_buf, UNIT_ATTENTION,
                                          ASC_CAPACITY_DATA_HAS_CHANGED,
                                          NULL);
                dev->flags &= ~TCMU_CAPACITY_CHANGED;
                return ret;
        }